homeassistant/components/homeassistant/triggers/time.py

```diff
@@ -187,10 +222,8 @@ def update_entity_trigger(entity_id: str, new_state: State | None = None) -> Non
                 )
             )
 
-    # Track state changes of any entities.
-    removes.append(
-        async_track_state_change_event(hass, to_track, update_entity_trigger_event)
-    )
+    # Track state changes of all entities
```

Comment:
If we're anyway changing the comment, can we make it less misleading? We're not tracking all entities.